### PR TITLE
fix: adding fixed header

### PR DESCRIPTION
**Pull request by aballerr**
https://uniswaplabs.atlassian.net/browse/WEB-1893

Made header on list sticky

Screen Shot 2022-12-16 at 11 30 12 AM

### CODE REVIEW

File: src/nft/components/profile/list/NFTListingsGrid.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const NFTListingsGrid = ({ selectedMarkets }: { selectedMarkets: ListingM
 
   return (
     <Column>
-      <Row marginTop="20">
+      <Row position="sticky" top="72" paddingY="24" zIndex="1" backgroundColor="backgroundBackdrop">
```
Comment (grabbou) [inline src/nft/components/profile/list/NFTListingsGrid.tsx:47]: I know this is still a draft, but shouldn't we migrate to Styled Components as we tweak different components?

Comment (aballerr) [reply src/nft/components/profile/list/NFTListingsGrid.tsx:47]: if I was adding a new component i would, but realistically this entire file should be rewritten.  Rewriting one line seems little value add here

Comment (cbachmeier) [reply src/nft/components/profile/list/NFTListingsGrid.tsx:47]: I agree with Mike here since we're already here and this will save time later on

Comment (cbachmeier) [reply src/nft/components/profile/list/NFTListingsGrid.tsx:47]: nit: Can you use one of the shared style zIndices?

**Review by cbachmeier — APPROVED** [review]
LGTM minor nits

**Pull request merged**